agent/engine/docker_task_engine.go

```diff
@@ -508,6 +509,20 @@ func (engine *DockerTaskEngine) GetTaskByArn(arn string) (*api.Task, bool) {
 }
 
 func (engine *DockerTaskEngine) pullContainer(task *api.Task, container *api.Container) DockerContainerMetadata {
+	if container.Type == api.ContainerCNIPause {
+		// ContainerCNIPause images are managed at startup
+		return DockerContainerMetadata{}
+	}
+	switch container.Type {
+	case api.ContainerCNIPause:
+		// ContainerCNIPause image are managed at startup
+		return DockerContainerMetadata{}
+	case api.ContainerEmptyHostVolume:
+		// ContainerEmptyHostVolume image is either local (must be imported) or remote (must be pulled)
+		if emptyvolume.LocalImage {
+			return engine.client.ImportLocalEmptyVolumeImage()
```

Comment:
Did `create` better describe what was happening behind the scenes here?

Reply:
`Import` is the verb that is used in the Docker API.